### DISCUSSION TITLE
Change layout of the cards for research page

### DISCUSSION THREAD

**Discussion by Srihitha-l** [greenelab/lab-website-template#202] [Q&A] [answered]
How do I change what the layout of each publication card on the research page looks like? Not just the CSS styling, but what information is included in it.

**Comment by vincerubinetti** [comment]
The card component is styled with css in `_styles/citation.scss`. It takes information/parameters that are explained [here](https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/citation). If you are using the research page that comes with the template, it is using the list component and reading those parameters from `_data/citations.yaml` which gets generated from your input `sources.yaml`, `orcid.yaml`, etc.

You'll have to be more specific if you'd like more specific help.

**Reply by Srihitha-l** [reply]
I want to change the information listed on the card . For example,  I don't want the publisher's name to appear in the lower-left corner of the card. But if I leave that parameter blank, it defaults to 'no publisher info'. I also do want to change the colors on the card, but if I change it in the _styles/card.scss, it will change it for the cards on the blog page too.

**Reply by vincerubinetti** [reply, marked as answer]
I think you're referring to the "citation" component, not the "card" component. I edited my comment above to reflect that.

You can remove the publisher info (or just the default "[no info]") from the citation component on this line:
https://github.com/greenelab/lab-website-template/blob/main/_includes/citation.html#L57

You can edit the citation styles in this file:
https://github.com/greenelab/lab-website-template/blob/main/_styles/citation.scss

If you're using the citation component on the blog page as well, yes, changing that will affect the citation component everywhere... If you need to have it look different in different places, that's beyond the scope of what the template provides, and you'll have to add a custom parameter to the citation component include.

**Comment by Srihitha-l** [comment]
Yes, I meant the citation, thank you so much!
This is what I was looking for
